style.css

```diff
@@ -0,0 +1,271 @@
+body {
+  display: grid;
+  grid-template:
+    "header header header" 50px
+    "left-bar main right-bar" 1fr /
+    200px 1fr 200px;
+  height: 100vh;
+  font-family: sans-serif;
+  margin: 0;
+  background: black;
+  color: white;
+  margin-left: 3vw;
+  margin-right: 5vw;
+}
+
+
+/* Header start */
+.header {
+  grid-area: header;
+  display: grid;
+  grid-template: 
+    "img h1 search-bar" 50px /
+   200px 1fr 200px;
+  position: sticky; top: 0;
+}
+
+.header img {
+  width: 60px;
+}
+
+.header h1 {
+  border-left: solid 1px #2F3336;
+  border-right: solid 1px #2F3336;
+  font-size: large;
+  margin-left: 10px;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-top: 10px;
+  padding-left: 10px;
+}
+
+.header .search-bar {
+  display: inline-flex;
+  place-items: center;
+  background: #202327;
+  border-radius: 50px;
+  margin: 10px;
+  width: 225px;
+}
+
+.header .search-bar p {
+  margin: 0;
+  color: #71767B;
+  text-align: center;
+  line-height: 2;
+}
+
+.header .search-bar img {
+  width: 20px;
+  margin: 15px;
+}
+/* header end */
+
+
+/* twitter main start */
+.main {
+  border-top: solid 1px #2F3336;
+  border-left: solid 1px #2F3336;
+  border-right: solid 1px #2F3336;
+  height: 100vh;
+  padding-right: 10px;
+  margin-left: 10px;
+}
+
+.enter-tweet .text-box {
+  display: inline-flex;
+  margin: 20px;
+  gap: 50px;
+  color: #6A6F74;
+  line-height: 60px;
+}
+
+.tweets {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.tweet-cards {
+  border-top: solid 1px #2F3336;
+  display: grid;
+  grid-template:
+    "pfp1 name tag time" 50px
+    "message message message message" 100px /
+    60px 100px 100px 1fr 100px;
```

Comment:
You have 5 width amounts here but only 4 areas defined a line higher, resulting is an extra 100px empty space on the right of all tweets.